### DISCUSSION TITLE
Git Extensions refuse to start

### DISCUSSION THREAD

**Discussion by IO5** [gitextensions/gitextensions#11096] [Q&A]
### Environment

- GitExtensions version: 4.1.0
- GIT version: 2.41.0.windows.1
- OS version: Win10 22H2
- .NET version:  4.8.04084

### Issue description

When I try to run git extensions I'm greeted with a prompt:
![image](https://github.com/gitextensions/gitextensions/assets/1437195/21389539-a7c7-4911-bed3-759615c0c7b0)
appearing 3 times in a row and then
![image](https://github.com/gitextensions/gitextensions/assets/1437195/ad1267d9-e742-4f75-ae00-5feae37cbb9a)
when I point it to `where git` location I get the first prompt 3 more times and then process exits.
The issue started on a earlier version, but since I updated git and did a clean reinstall of git extensions with %appdata% %localappdata% folders deleted. Did not help.
What's extra weird I had no issues before, and I did not mess with git, git extensions updates/config before the issues appeared. 

### Steps to reproduce

running GitExtensions.exe

### Did this work in previous version of GitExtensions?

I had 4.0.0 installed before

### Diagnostics

can't even get to the command log

**Comment by vbjay** [comment]
Did you think you might want to upgrade git to be current?

https://git-scm.com/downloads

It looks like you are pointing to an older version of git and not the current version. Look at your path and what is output of where git.

**Comment by IO5** [comment]
> Did you think you might want to upgrade git to be current?

I did
`git --version`
`git version 2.41.0.windows.1`
from both cmd and bash

> It looks like you are pointing to an older version of git and not the current version. Look at your path and what is output of where git.

`C:\Program Files\Git\cmd\git.exe`

looks right to me

After digging some more it might be issue with git itself after all. git bash and git itself appears to be working fine, but gitk chokes on empty $git_version also.

**Comment by gerhardol** [comment]
Set command to `C:\Program Files\Git\bin\git.exe`

**Comment by IO5** [comment]
Ok, I fugerd out the root cause: buggy antivirus. The problem disappeared after next av update.

**Reply by vbjay** [reply]
Which av?
